src/enum_clike.rs

```diff
@@ -0,0 +1,56 @@
+//! lint on clike enums that are repr(isize/usize) and have values that don't fit into an i32
+
+use rustc::lint::*;
+use syntax::ast::{IntTy, UintTy};
+use syntax::attr::*;
+use rustc_front::hir::*;
+use rustc::middle::const_eval::{ConstVal, EvalHint, eval_const_expr_partial};
+use rustc::middle::ty;
+use utils::span_lint;
+
+/// **What it does:** clike enums that are repr(isize/usize) and have values that don't fit into an i32
+///
+/// **Why is this bad?** This will truncate the variant value on 32bit architectures, but works fine on 64 bit
+///
+/// **Known problems:** None
+///
+/// **Example:** #[repr(usize)] enum NonPortable { X = 0x1_0000_0000, Y = 0 }
```

Comment:
You’re missing code block here, also for `i32`. *clike* should probably be *C-like*.